deploy/osd-delete-backplane-script-resources/20-delete-backplane-script-resources.CronJob.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: osd-delete-backplane-script-resources
+  namespace: openshift-backplane-managed-scripts
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "* * */1 * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          affinity:
+            nodeAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+              - preference:
+                  matchExpressions:
+                  - key: node-role.kubernetes.io/infra
+                    operator: Exists
+                weight: 1
+          tolerations:
+            - effect: NoSchedule
+              key: node-role.kubernetes.io/infra
+              operator: Exists
+          restartPolicy: Never
+          serviceAccountName: osd-delete-backplane-script-resources-sa
+          containers:
+            - name:  osd-delete-backplane-script-resources
+              imagePullPolicy: Always
+              image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+              args:
+                - /bin/bash
+                - -c
+                - |
+                  set -euxo pipefail
+
+                  NS="openshift-backplane-managed-scripts"
+                  LABEL="managed.openshift.io/backplane-job-uuid"
+
+                  ### Gather data
+                  # Format: pod_name pod_status
+                  PODS=$(oc get pod -n $NS --selector $LABEL -o custom-columns=NAME:.metadata.name,PHASE:.status.phase --no-headers)
+
+                  # Format: pod_name(running)
+                  PODS_RUNNING=$(echo "$PODS" | { grep -i Running || test $? = 1; } | awk '{print $1}')
```

Comment:
```                   
PODS_RUNNING=$({ grep -i Running || test $? = 1; } <<<"$PODS" | awk '{print $1}')
```

Reply:
use `test $? = 1` here in case `grep` exits with 2, which could be other errors than no match.